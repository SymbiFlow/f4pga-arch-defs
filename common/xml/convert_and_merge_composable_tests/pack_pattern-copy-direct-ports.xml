<?xml version="1.0"?>
<!--
  Convert
     <direct XXXX><pack_pattern name="xxx"/>
  to
     <direct XXXX><pack_pattern name="xxx" XXXX/>
  -->
<xml>
	<interconnect>
  <direct input="SB_FF.D" name="VPR_FF-D" output="VPR_FF.D">
    <pack_pattern name="A"/>
  </direct>
  <direct>
   <port type="input" from="LUTFF" name="FCIN"/><port type="output" from="SB_CARRY" name="CI"/>
   <pack_pattern name="CARRYCHAIN"/>
  </direct>
 </interconnect>
</xml>
