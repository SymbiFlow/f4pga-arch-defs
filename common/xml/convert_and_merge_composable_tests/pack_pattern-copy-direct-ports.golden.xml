<?xml version="1.0"?>
<xml>
  <interconnect>
    <direct input="SB_FF.D" name="VPR_FF-D" output="VPR_FF.D">
      <pack_pattern in_port="SB_FF.D" name="A" out_port="VPR_FF.D"/>
    </direct>
    <direct input="LUTFF.FCIN" name="SB_CARRY-CI" output="SB_CARRY.CI">
      <pack_pattern in_port="LUTFF.FCIN" name="CARRYCHAIN" out_port="SB_CARRY.CI"/>
    </direct>
  </interconnect>
</xml>
